src/layout/Menu/Menu.style.ts

```diff
@@ -153,18 +143,54 @@ export const MenuStyled = styled.div<{
 
 export const SubMenu = styled.div<{ total: number }>`
   width: 100%;
-  padding-top: 0.25rem;
 
   > div {
-    animation: ${fade} 120ms ease-in-out both;
+    animation: ${fade} 140ms ease-in-out both;
 
     ${(p) =>
       [...new Array(30)].map(
         (_, i) => css`
-          &:nth-child(${i}) {
-            animation-delay: ${i * (20 + 100 / p.total)}ms;
+          &:nth-child(${i + 1}) {
+            animation-delay: ${(i + 1) * (20 + 100 / p.total)}ms;
           }
         `
       )};
   }
 `;
+
+export const Wrapper = styled.div<{
+  $height?: number;
+}>`
+  border-radius: 0.2rem;
+  position: relative;
+  transition: 80ms ease-in-out;
+
+  & ${Toggle}:hover + ${ItemStyled} {
+    background: ${({ theme }) =>
+      theme.name === 'dark' ? grayscale(700) : slate(100)};
+  }
+
+  ${(p) =>
+    p.$height &&
+    css`
+      transition: 230ms ease-in-out;
+      height: ${(p.$height + 1) * 2.5 + 0.75}rem;
```

Comment:
This may be worth moving out to a `style` prop to prevent excess class generation now that menus can be more complex.

Reply:
It looks like this code isn't actually doing anything since `$height` will always be false.